Triangular_Number/Triangular_Number.c

```diff
@@ -0,0 +1,42 @@
+/*A number is termed as triangular number if we can represent it in the form of triangular grid of 
```

Comment:
Please change the description of problem. Exactly the same thing is written on Geeks for Geeks. 
Also add more comments which illustrate the usage of this equation and highlight your understanding of the problem by doing so.